paper/src/main/java/com/denizenscript/denizen/paper/events/EntityKnockbackByEntityScriptEvent.java

```diff
@@ -0,0 +1,133 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.scripts.containers.ScriptContainer;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.destroystokyo.paper.event.entity.EntityKnockbackByEntityEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.util.Vector;
+
+public class EntityKnockbackByEntityScriptEvent extends BukkitScriptEvent implements Listener  {

```

Comment:
Stray doublespace ^
You should enable automatic formatting in IntelliJ btw, helps with this stuff